locales/lv/messages.json

```diff
@@ -51,6 +51,7 @@
     },
     "autoConnectEnabled": {
         "message": "Automātiskā savienošanās: Ieslēgta - Konfigurators mēģinās automātiski savienoties, kad tiks detektēts jauns ports"
+        "description": "Edited in crowdin by EGParadox."
```

Comment:
This needs a comma at the end of the previous line or it will break the JSON format.